packages/mgt-element/src/providers/Providers.ts

```diff
@@ -240,12 +238,13 @@ export class Providers {
 
   private static _globalProvider: IProvider;
   private static _cacheId: string;
-  private static _mePromise: Promise<User>;
+  private static _me: User; // NOTE: the prior implementation of _mePromise meant that the promise was cached as
+  // fulfilled with null for the user and was never updated if the /me call failed.
```

Comment:
if the /me call fails now, does it get retried? I don't understand how that is addressed here.

Reply:
All of them don't, for instance, the very last one doesn't.

Reply:
line 109 would get null and then then next time through 108 would still be TRUE and it would try again...

<img width="396" alt="image" src="https://github.com/plasne/microsoft-graph-toolkit/assets/15094473/3c166de0-3a19-4727-baa2-d7c98ce99905">